engine/src/main_loop.cpp

```diff
@@ -792,22 +791,27 @@ void createObjects(std::vector<std::string> &fighter0name,
     vector<std::string> fighter0mods;
     vector<int> fighter0indices;
 
-    static Vector TerrainScale(game_options()->xscale, game_options()->yscale, game_options()->zscale);
+    static Vector * terrain_scale;
```

Comment:
Why is this code better? Vector isn't a heavy class. It's really three floats.

Reply:
I suppose there are other options as well, but those seem like the two best options. Feel free to suggest something else if you like.